### DISCUSSION TITLE
httpd_txrx: httpd_sock_err: error in recv : 104

### DISCUSSION THREAD

**Discussion by Sven4597** [jomjol/AI-on-the-edge-device#2722] [Q&A]
Hello, I cannot access the web interface with the IP. The log always contains the following error httpd_txrx: httpd_sock_err: error in recv: 104
I have already bought a new SD card and a new ESP Cam but the error remains the same.

What am I doing wrong?

**Comment by friedpa** [comment]
Can you please provide the LOG (set LOG Level before to Debug)

**Comment by Sven4597** [comment]
[esp-web-tools-logs (2).txt](https://github.com/jomjol/AI-on-the-edge-device/files/13514268/esp-web-tools-logs.2.txt)

**Comment by caco3** [comment]
>  httpd_txrx: httpd_sock_err: error in recv : 104

This is a non-critical issue and is not related to your issue to not be able to access the web UI, see https://github.com/jomjol/AI-on-the-edge-device/issues/1869

**Comment by caco3** [comment]
Can you ping the device (`ping <IP>`)?

Can you access some non-SD-card URLs like `http://<IP>/rssi` or `http//<IP>/value`?

**Comment by Sven4597** [comment]
Das einrichten über den Handy Browser funktioniert ohne Fehler.
Wenn ich am PC aufrufe geht es nicht.
